cadvisor.go

```diff
@@ -76,6 +76,10 @@ func (ml *metricSetValue) String() string {
 }
 
 func (ml *metricSetValue) Set(value string) error {
+	ignoreMetrics = metricSetValue{}
```

Comment:
I think this might cause a panic if `--disable_metrics` is not empy:

```
    goroutine 1 [running]:
    panic(0xb0c8c0, 0xc82020cdd0)
    	/usr/local/go/src/runtime/panic.go:481 +0x3e6
    main.(*metricSetValue).Set(0x15ac528, 0x7fffe9354f34, 0x4, 0x0, 0x0)
    	/go/src/github.com/google/cadvisor/cadvisor.go:85 +0x1da
    flag.(*FlagSet).parseOne(0xc820066060, 0xc820118801, 0x0, 0x0)
    	/usr/local/go/src/flag/flag.go:881 +0xdd9
    flag.(*FlagSet).Parse(0xc820066060, 0xc820064110, 0x3, 0x3, 0x0, 0x0)
    	/usr/local/go/src/flag/flag.go:900 +0x6e
    flag.Parse()
    	/usr/local/go/src/flag/flag.go:928 +0x6f
    main.main()
    	/go/src/github.com/google/cadvisor/cadvisor.go:99 +0x68
    panic: assignment to entry in nil map
```

I'm unsure about the details. I'm not very familiar with Go.